dataprofiler/profilers/graph_profiler.py

```diff
@@ -330,12 +330,12 @@ def _update_categorical_distribution(
     @BaseColumnProfiler._timeit(name="num_nodes")
     def _get_num_nodes(self, graph: nx.Graph) -> int:
         """Compute the number of nodes."""
-        return graph.number_of_nodes()
+        return cast(int, graph.number_of_nodes())
```

Comment:
As far as I could tell, the `networkx` module doesn't have type annotations for graphs.

Reply:
fair